.gitmodules

```diff
@@ -32,3 +32,7 @@
 	path = cloud-provider-azure
 	url = https://github.com/openshift/cloud-provider-azure
 	branch = release-4.14
+[submodule "csi-proxy"]
```

Comment:
Please add this to [HACKING.md#updating-git-submodules](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/HACKING.md#updating-git-submodules)